Check broken links (#1354)
ocaml/v3.ocaml.org: data/changelog/odoc/2019-03-08-odoc-1.4.0.md

  - Paragraph headings (`{4 ...}`) and subparagraph headings (`{5 ...}`) (#217,
    Rizo Isrof).
  - `odoc support-files-targets` command (#232).

>   - Recommend [`bsdoc`](https://ostera.github.io/bsdoc/docs/BsDoc/ - [404 Not Found]) for using

Reviewer: ```suggestion
  - Recommend [`bsdoc`](https://rescript-lang.org/blog/bucklescript-is-rebranding) for using
```
BsDoc no longer available on Ostera's current GitHub page. This is the best I could find about it.

Reviewer 2: I found this:

```suggestion
  - Recommend [`bsdoc`](https://github.com/reasonml-community/bsdoc) for using
```

>     odoc with BuckleScript (#269, Leandro Ostera).

Reviewer: ```suggestion
    `odoc` with BuckleScript (#269, Leandro Ostera).
```

We are pleased to announce release **1.4.0** of [**odoc**](https://github.com/ocaml/odoc), the new, Dune-friendly documentation generator for OCaml and ReasonML.

> Some sample output, which you can browse [here](http://odis.io/odoc-demo/order/Order/index.html - [404 Not Found]):

Reviewer: ```suggestion

```
That site no longer exists

> In summary, compared with 1.2.0, odoc now has:

Reviewer: ```suggestion
In summary, compared with 1.2.0, `odoc` now has:
```

> - [New appearance](http://odis.io/odoc-demo/order/Order/index.html - [404 Not Found]).

Reviewer: ```suggestion
- [New appearance](https://ocaml.github.io/odoc/#overview).
```
Site no longer exists. Perhaps a link to the `odoc` website?

Reviewer 2: Could not find a link on the Wayback Machine that captures the new look. I only get pages without CSS.

```suggestion
- New appearance.
```

- Tables of contents on each page.
- [Generation of docs in ReasonML syntax](https://github.com/ocaml/odoc/pull/156).

> - [Interop with BuckleScript](https://ostera.github.io/bsdoc/docs/BsDoc/ - [404 Not Found]).

Reviewer: ```suggestion
- [Interop with BuckleScript](https://rescript-lang.org/blog/bucklescript-is-rebranding).
```
best think I could find about BuckleScript

Reviewer 2: The project to search for was BsDoc.

```suggestion
- [Interop with BuckleScript](https://github.com/reasonml-community/bsdoc).
```
